openfisca_web_api/loader/entities.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals, print_function, division, absolute_import
+from openfisca_core.commons import to_unicode
+
+
+def build_entities(tax_benefit_system):
+    entities = {
+        entity.key: build_entity(entity)
+        for entity in tax_benefit_system.entities
+        }
+    return entities
+
+
+def build_entity(entity):
+    split_doc = to_unicode(entity.doc).split("\n")
+    formatted_start_doc = split_doc[1:] if split_doc[0] == "" else split_doc
+    formatted_end_doc = formatted_start_doc[:-1] if formatted_start_doc[-1] == "" else formatted_start_doc
+    formatted_doc = "\n".join(formatted_end_doc)
```

Comment:
I think you can just use `formatted_doc  = to_unicode(entity.doc.strip())`

`strip` removes all trailing white spaces, and `\n` is considered as a particular white space 🎉

Reply:
... 🤦